src/commands/appdistribution-group-create.ts

```diff
@@ -6,6 +6,7 @@ import { getProjectName } from "../appdistribution/options-parser-util";
 
 export const command = new Command("appdistribution:group:create <displayName> [alias]")
   .description("create group in project")
+  .alias("appdistribution:groups:create")
```

Comment:
Should this be the canonical form and `group:*` the alias?

What would the effect be? I guess we don't want both forms to show up in the help text - only the plural form. Or at least give the plural form more prominence.